src/App.jsx

```diff
@@ -1,44 +1,33 @@
-import React from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import { useState } from 'react';
+import { Tabs } from './components/Tabs/Tabs';
```

Comment:
Ensure that the `Tabs` component is correctly implemented in the `./components/Tabs/Tabs` path and that it accepts the props `tabs`, `activeTabId`, and `onTabSelected` as expected. If there are issues with the component, it might not render or function correctly.